drivers/sony-bravia-android-tv/device.js

```diff
@@ -10,19 +9,34 @@ class SonyBraviaAndroidTvDevice extends Homey.Device {
 
     console.log(`${this.data.name} initialized.`);
 
-    this.registerTasks();
-    this.registerFlowListeners();
-    this.registerCapabilityListeners();
+    this.setCapabilityListeners();
+    this.checkDeviceInterval(this.data.settings.polling);
+

```

Comment:
Just very minor, but have a leftover newline here 😄 Should you touch this file by any chance,...